marklogic/assets/recommended_monitors/marklogic_high_load.json

```
@@ -0,0 +1,28 @@
+{
+	"name": "[MarkLogic] Forest Processing Load is High on {{host.name}}",
+	"type": "query alert",
+	"query": "avg(last_5m):avg:marklogic.forests.total_load{*} > 2000000000",
```

```suggestion
	"query": "avg(last_5m):avg:marklogic.forests.total_rate{*} > 2000000000",
```
Actually `total_load` is in second per second (I will fix this typo), so let's use `total_rate`